AirportBooking/Repositories/BookingRepository.cs

```
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using AirportBooking.Models;
+
+namespace AirportBooking.Repositories
+{
+    public class BookingRepository : IBookingRepository
+    {
+        private readonly string _filePath = "data/bookings.json";
+
+        private async Task<List<Booking>> ReadBookingsAsync()
+        {
+            if (!File.Exists(_filePath))
```

we should throw an exception, since file path should be valid